docs/source/depth/unit_parsing.rst

```diff
@@ -7,8 +7,13 @@ By default, Pint supports a larger set of units than the Citrine Platform.
 Therefore, we include a custom unit definition file in gemd-python: `citrine_en.txt`_.
 This file contains the most commonly used units and will grow over time.
 
+In support of common patterns in materials science, we permit including scaling factors in a unit of measure.
+For example, industrial researchers may have recorded historical data as ``g / 2.5 cm``.
+While this could be converted a simple SI expression, that would prevent researchers from representing the data
+as originally reported, thus creating a potential source of error during the input process.
+
 Requests for support of additional units can be made by opening an issue in the `gemd-python repository`_ on github.
 
-.. _Pint: https://pint.readthedocs.io/en/0.9/
+.. _Pint: https://pint.readthedocs.io/en/stable/
```

Comment:
If our docs are gonna point to a dependency's docs, shouldn't they point to the version we currently use? Since they're more likely to not apply if we just always say "stable".

Reply:
Updated to the version currently in requirements.txt.  Another piece that will require continued attention.